### DISCUSSION TITLE
如何获取群里面的人撤回的消息

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#114] [Q&A] [answered]
### 涉及的编程语言

Java

### 疑问描述

如何获取群里面的人撤回的消息呢

**Comment by ForteScarlet** [comment, marked as answer]
https://github.com/simple-robot/simpler-robot/issues/469

**Reply by yujumpjump** [reply]
设置好了配置  通过什么方法来获取呢 有相关的api吗

**Reply by ForteScarlet** [reply]
[MiraiXxxRecallMessageEvent](https://docs.simbot.forte.love/components/mirai/simbot-component-mirai-core/love.forte.simbot.component.mirai.event/-mirai-group-message-recall-event/index.html)
